### PR TITLE
Allow output of mean of biased n(z)

### PR DESCRIPTION
**Pull request by tilmantroester**
This PR adds the mean of the shifted n(z) as a convenience derived parameter for compatibility with the KiDS-1000 setup.

### CODE REVIEW

**Comment by joezuntz** [comment]
Ignore the test failures; they are unrelated and a symptom of the usual python package management clusterfuck.

**Review by joezuntz — CHANGES_REQUESTED** [review]
Minor change request - otherwise looks good.

File: number_density/photoz_bias/module.yaml [hunk 1]
```diff
@@ -50,6 +50,14 @@ params:
         meaning: Whether to use one value per bin, If False, use one value for all bins.
         type: bool
         default: true
+    output_deltaz:
```
[the remark this answers is not in the document]
Comment (tilmantroester) [reply number_density/photoz_bias/module.yaml]: Done.

**Review by joezuntz — APPROVED** [review]

**Comment by joezuntz** [comment]
Looks like the failing tests this time are real:
https://github.com/joezuntz/cosmosis-standard-library/actions/runs/5512520274/jobs/10049458482?pr=84

**Pull request merged**